src/components/views/SingleArticle.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Container, Divider } from "semantic-ui-react";
+import GetRates from "../containers/rating/GetRates";
+import RateArticle from "../containers/rating/RateArticle";
+
+const SingleArticle = props => (
```

Comment:
do we need this file?

Reply:
we don't need it was temporary to ensure my feature works